# Time of sensor data acquisition, and the coordinate frame ID
Header header

# Can message ident 0x19F8051D

uint8 SECONDS
uint8 MINUTES
uint8 HOURS
uint8 DAY
uint8 MONTH
uint16 YEAR

float64 Latitude
float64 Longitude
float64 Altitude
uint8 TypeOfSystem
uint8 GNSSMethod
uint8 GNSSIntegrity
float64 HDOP
float64 PDOP
uint8 NumberOfSatelites
uint8 NumberOfRefStations